src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Unit.groovy

```diff
@@ -0,0 +1,13 @@
+package life.qbic.datamodel.dtos.imaging.properties
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
```

Comment:
Unsure if we need it. We need to check with @luiskuhn , what meaningful ranges and units are for a position.

Reply:
Good point, I also was a bit confused. Lets double check 👍